.storybook/blocks/component-details.jsx

```diff
@@ -15,12 +15,45 @@ import "./component-details.css";
 const ComponentDetails = () => {
 	const storyMeta = useOf('meta');
 	const versionNumber = storyMeta?.csfFile?.meta?.parameters?.componentVersion ?? "Unavailable";
+  
+  const packageName = storyMeta?.csfFile?.meta?.parameters?.componentPkgName ?? undefined;
+  const packageLink = (packageName && typeof packageName !== "undefined") ? `https://npmjs.com/${packageName}` : false;
+  
+  const packageGitHubName = packageLink ? packageName.split('/').pop() : undefined;
```

Comment:
I think we can use the `packageName` from the `package.json` and just split it after the `@spectrum-css` namespace's slash. I'm like...99% sure that our component dir names match our npm package names.